deployment/roles/reverseproxy/templates/security-headers.toml.j2

```diff
@@ -0,0 +1,35 @@
+[tls.options]
+    [tls.options.default]
+      minVersion = "VersionTLS12"
+      sniStrict = true
+      cipherSuites = [
+          "TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256",
+          "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256",
+          "TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384",
+          "TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305",
+          "TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305",
+          "TLS_AES_128_GCM_SHA256",
+          "TLS_AES_256_GCM_SHA384",
+          "TLS_CHACHA20_POLY1305_SHA256",
+          "TLS_FALLBACK_SCSV"
+      ]
+
+[http.middlewares]
+    [http.middlewares.security-headers.headers]
+      # CORS
```

Comment:
Hm, CORS should be specified on application level, not on server level, right?

Reply:
It's not the server level, this is a middleware that you can activate for your service. 
So its basically the run configuration of your service.